bitnami/prometheus-operator/templates/prometheus/prometheus.yaml

```diff
@@ -40,7 +40,9 @@ spec:
   {{- else if .Values.prometheus.replicaExternalLabelName }}
   replicaExternalLabelName: "{{ .Values.prometheus.replicaExternalLabelName }}"
   {{- end }}
-  {{- if and .Values.prometheus.ingress.enabled .Values.prometheus.ingress.hosts }}
+  {{- if .Values.prometheus.externalUrl }}
+  externalUrl: "{{ .Values.prometheus.externalUrl }}"
+  {{- else if and .Values.prometheus.ingress.enabled .Values.prometheus.ingress.hosts }}
```

Comment:
You need to add this `prometheus. externalUrl ` parameter to the **values.yaml** and the **values-production.yaml**